Gems/PhysX/Code/Source/Articulation.cpp

```diff
@@ -62,50 +62,55 @@ namespace PhysX
 
     void ArticulationLink::AddCollisionShape(const ArticulationLinkData& thisLinkData)
     {
-        const Physics::ColliderConfiguration* colliderConfiguration = thisLinkData.m_shapeColliderConfiguration.first.get();
-        const Physics::ShapeConfiguration* shapeConfiguration = thisLinkData.m_shapeColliderConfiguration.second.get();
-
-        if (shapeConfiguration && colliderConfiguration)
+        m_physicsShapes.clear();
+        for (auto& [colliderConfiguration, shapeConfiguration] : thisLinkData.m_shapeColliderConfigurationList)
         {
-            if (shapeConfiguration->GetShapeType() == Physics::ShapeType::PhysicsAsset)
+            if (shapeConfiguration && colliderConfiguration)
             {
-                const auto* physicsAssetShapeConfiguration =
-                    static_cast<const Physics::PhysicsAssetShapeConfiguration*>(shapeConfiguration);
-                if (!physicsAssetShapeConfiguration->m_asset.IsReady())
+                if (shapeConfiguration->GetShapeType() == Physics::ShapeType::PhysicsAsset)
                 {
-                    const_cast<Physics::PhysicsAssetShapeConfiguration*>(physicsAssetShapeConfiguration)->m_asset.BlockUntilLoadComplete();
+                    const auto* physicsAssetShapeConfiguration =
+                        static_cast<const Physics::PhysicsAssetShapeConfiguration*>(shapeConfiguration.get());
+                    if (!physicsAssetShapeConfiguration->m_asset.IsReady())
+                    {
+                        auto asset_status = const_cast<Physics::PhysicsAssetShapeConfiguration*>(physicsAssetShapeConfiguration)
+                                                ->m_asset.BlockUntilLoadComplete();
+                        AZ_Error(
+                            "PhysX",
+                            asset_status == AZ::Data::AssetData::AssetStatus::Ready,
+                            "Failed to load physics asset %s ",
+                            physicsAssetShapeConfiguration->m_asset.GetHint().c_str());
+                        if (asset_status != AZ::Data::AssetData::AssetStatus::Ready)
```

Comment:
The `asset_status` name should be lowerCamelCase
```suggestion
                        auto assetStatus = const_cast<Physics::PhysicsAssetShapeConfiguration*>(physicsAssetShapeConfiguration)
                                                ->m_asset.BlockUntilLoadComplete();
                        AZ_Error(
                            "PhysX",
                            assetStatus == AZ::Data::AssetData::AssetStatus::Ready,
                            "Failed to load physics asset %s ",
                            physicsAssetShapeConfiguration->m_asset.GetHint().c_str());
                        if (assetStatus != AZ::Data::AssetData::AssetStatus::Ready)
```